R/revolver_dumpStatistics.R

```diff
@@ -0,0 +1,65 @@
+#' Dump statistics for fit to an Excel file.
+#'
+#' @param x REVOLVER cohort fit
+#' @param file output file
+#'
+#' @return invisibly, the list of data.frames written to the Excel file 
+#' 
+#' @importFrom writexl write_xlsx
+#' @importFrom pio pioHdr pioTit
+#'
+#' @examples
+#' data(CRC.cohort)
+#' fit <- revolver_fit(CRC.cohort)
+#' revolver_dumpStatistics(fit)
+#'
+#' @export
+revolver_dumpStatistics <- function(x, file='REVOLVER-Statistics.xlsx') {
+
+  pioHdr('REVOLVER: Dump statistics to Excel', "File: ", file, suffix='\t')
+  pioTit('Features: drivers, occurrences & information transfer')
+  ft <- revolver.featureMatrix(x)
+
+  # helper fn for data.frame'ing
+  dfWithRownames <- function(x) {
+    if (is(x, "matrix")) {
+      return(cbind.data.frame(Name=rownames(x), as.data.frame(x)))
+    } else { 
+      return(data.frame(Name=names(x), Value=x))
+    }
+  }
+
+  sheets <- list()
+  sheets[["Drivers"]] <- x$dataset[x$dataset$is.driver, -1] # wtf is Misc?
```

Comment:
We are not sure that the first colum is always the one that shoudl be removed. It is important to keep also this column because is part of the input and should be left as it is.

Reply:
The `Misc` field stands for Miscellanea, I thought it was a reasonable free-format field to add to the data, where you can store whatever you want as annotation. I would dump it and more important would not assume that is necessarily column number 1.

Reply:
Misc is once again included in the output as requested